Source/Core/Core/HW/WiimoteReal/WiimoteReal.cpp

```diff
@@ -490,6 +489,10 @@ void WiimoteScanner::ThreadFunc()
           TryToConnectBalanceBoard(found_board);
       }
     }
+    else
+    {
+      Update();  // Does stuff needed to detect disconnects on Windows
```

Comment:
Disconnects won't be detected unless a game is running, which is less than ideal imo, but this was the old behaviour and it's still probably better than unloading/loading a Bluetooth DLL several times per second, until the Windows IO code is improved.